portfolio/src/main/webapp/script.js

```diff
@@ -11,18 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function getRandomQuoteUsingArrowFunctions() {
```

Comment:
Could you:

1. make this an arrow function too?
2. make the name reflect that this function adds to the page. (usually naming a fn `getX()` means it returns `x`)
3. remove the implementation details about arrow fns from the function name
4. add function documentation

```
/**
 * Adds a random quote from the server to the page.
 */
const addQuoteToPage = () => {
  // ...
};
```